### DISCUSSION TITLE
Makefile

### DISCUSSION THREAD

**Discussion by Maximus135** [dvarrazzo/py-setproctitle#93] [General]
Hello! 
I am extremely sorry to distract you, but I got a university task to work with your repository. The exact file I have to think about is Makefile. I can't find a line that actually builds the project. Could you please explain it?
Best wishes,
Maxim.

**Comment by dvarrazzo** [comment]
Hello,

This is a pretty much standard python project, so building, packaging, etc. goes through the `setup.py`. For a few tasks you can just call it with `python setup.py`, using the version of python you want to use, in case you have more on your system. Adding ``--help`` and ``--help-commands`` will give more information.

For instance, in order to build the project for installation on the local machine:

```
python setup.py build
sudo python setup.py install
```

in order to build the project locally, for development, you can run

```
python setup.py develop
```

What is that you have to do? I won't do your homework but if I can help you with more info let me know.

**Reply by Maximus135** [reply]
Thank you very much)
